x-pack/plugins/ml/public/application/explorer/explorer_charts/explorer_chart_single_metric.js

```diff
@@ -470,7 +475,14 @@ export class ExplorerChartSingleMetric extends React.Component {
       // Update all markers to new positions.
       scheduledEventMarkers
         .attr('x', (d) => lineChartXScale(d.date) - LINE_CHART_ANOMALY_RADIUS)
-        .attr('y', (d) => lineChartYScale(d.value) - SCHEDULED_EVENT_SYMBOL_HEIGHT / 2);
+        .attr(
+          'y',
+          (d) =>
+            // Fallback with domain's min value if value is null
+            // To ensure event markers are rendered properly at the bottom of the chart
+            lineChartYScale(d.value ? d.value : lineChartYScale.domain()[0]) -
```

Comment:
same question about the explicit `null` check as above

Reply:
Resolving with solution as above.